image/preseed/preseed_linux.go

```diff
@@ -565,14 +592,14 @@ func runUC20PreseedMode(opts *preseedOpts) error {
 // Core20 runs preseeding of UC20 system prepared by prepare-image in prepareImageDir
 // and stores the resulting preseed preseed.tgz file in system-seed/systems/<systemlabel>/preseed.tgz.
 // Expects single systemlabel under systems directory.
-func Core20(prepareImageDir, preseedSignKey, aaFeaturesDir string) error {
+func Core20(prepareImageDir, preseedSignKey, aaFeaturesDir, sysfsOverlay string) error {
```

Comment:
we might want to introduce an options struct here

Reply:
I added all those options now into one struct. I also improved other internal struct and now all the internal options are stored there. So we have only two structures rather than a gazillion parameters passed around.